src/sentry/static/sentry/app/views/performance/transactionDetails/content.tsx

```diff
@@ -112,6 +114,8 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     const transactionName = event.title;
     const query = decodeScalar(location.query.query, '');
 
+    const hasQuickTraceView = organization.features.includes('trace-view-quick');
```

Comment:
Should this be both `trace-view-quick` and `trace-view-summary`

Reply:
Hm... that's a good point, without `trace-view-summary`, `trace-view-quick` wouldn't work anyways. I'll add it.